Fix HTML encoding of hotspot headers

otacke opened the pull request:
When merged in, will fix the hotspot headers being displayed HTML encoded in the tooltip.
Reported in https://github.com/h5p/h5p-image-hotspots/issues/78.

fnoks commented:
Thanks for your PR. Unfortunately, it also affects the header itself. You should probably move the convertion to inside the hotspot.setTitle function.

You could test with e.g: "It's a <script>tag</script>"

I am closing this, but please reopen when it is fixed :)

otacke commented:
@fnoks I changed that (and since keeping HTML tags seems to be important, removed HTML stripping) and updated the branch - but I cannot reopen a pull request :-D You'll have to do that, or I'll have to create a new one.

otacke commented:
@fnoks Just don't want to get this forgotten ;-)

otacke commented:
@fnoks beep boop!

otacke commented:
@fnoks "Give me a ping, Vasily. One ping only, please." - PING
